### ISSUE TITLE
Does this project support multi-category tracking？

### ISSUE THREAD

**Issue by chengdianJiang** [ifzhang/FairMOT#327]
Hi thanks for your sharing, I want to train this model with my data, people and car, Does this project support multi-category tracking？

**Comment by ifzhang** [comment]
Yes, you can refer to this project which uses FairMOT to support multi-category tracking:
https://github.com/CaptainEven/MCMOT

**Comment by chengdianJiang** [comment]
Thank a lot~
